app/models/setting/provisioning.rb

```diff
@@ -46,4 +55,11 @@ def validate_safemode_render(record)
       record.errors[:base] << _("Unable to disable safemode_render when bmc_credentials_accessible is disabled")
     end
   end
+
+  def self.default_global_templates
+    TemplateKind::PXE.map do |pxe_kind|
+      templates = Proc.new { { "" => "" }.merge Hash[ProvisioningTemplate.unscoped.of_kind(pxe_kind).map { |tmpl| [tmpl.name, tmpl.name] }] }
+      self.set("global_#{pxe_kind}", N_("Global default %s template that will not be affected by upgrades") % pxe_kind, nil, N_("Global default %s template") % pxe_kind, nil, { :collection =>  templates })
```

Comment:
I'd suggest changing the text a little so that it's more clear, that this is the template that gets deployed to the tftp proxies and not just the template that is skipped when upgrading.

How about?
"Global default %s template. This template gets deployed to all configured TFTP servers. It will not be affected by upgrades."